DependencyInjection/Configuration.php

```diff
@@ -54,12 +54,15 @@ public function getConfigTreeBuilder()
                 ->booleanNode('log_exceptions')
                     ->defaultFalse()
                 ->end()
-                ->booleanNode('log_deprecations')
-                    ->defaultFalse()
-                ->end()
                 ->booleanNode('log_commands')
```

Comment:
what's about moving `ignored_commands` inside this section? Same optimizations for `http`

Something like this (can be improved)
```yaml
ekino_new_relic:
    application_name: Awesome Application
    deployment_names: ~

    api_key:
    license_key:

    xmit: false
    logging: false
    instrument: false

    log_exceptions: false

    log_deprecations: false

    log_logs:
       channels: [app]
       level: error
       service: app.my_custom_handler
    
    log_commands:
        ignored_commands: []
    
    http:
        using_symfony_cache: false
        transaction_naming: route
        transaction_naming_service: ~
        ignored_routes: []
        ignored_paths: []
    

    twig: true
```